backend/entities/game.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, OneToMany } from 'typeorm';
+import { Session } from './session.entity';
+import { ParticipantScore } from './participant-score.entity';
+
+@Entity()
+export class Game {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  averageEstimationScore: number;
+
+  @ManyToOne(() => Session, session => session.games)
+  session: Session;
```

Comment:
why there is a need of ManyToOne relation
dont you think when we have oneToMany relation in session entity?

Reply:
@hamsof 
```
Yes, when using TypeORM to define a many-to-one relationship between two entities (let's say A and B), you typically need to specify the relationship on both entities.

In entity A, you'll define the many-to-one relationship using the @ManyToOne decorator, specifying the target entity (B in this case). This decorator should go on the property of entity A that represents the relationship with entity B.

In entity B, you'll define the inverse side of the relationship using the @OneToMany decorator. This decorator should also go on a property of entity B, representing the collection of entities A related to B.
```

Credits: ChatGPT

Reply:
yes I get it I mean we dont need ManyToOne for relationship its just navigating from inverse.

Like parent to child (one to many) but if u wanna navigate inverse then define
child to parent (many to one)